app/src/main/java/com/hamdan/forzenbook/domain/usecase/login/LoginRequestValidationUseCase.kt

```diff
@@ -0,0 +1,7 @@
+package com.hamdan.forzenbook.domain.usecase.login
+
+interface LoginRequestValidationUseCase {
+
+    suspend operator fun invoke(email:String):Int
```

Comment:
So help me I will add an automatic format checker to your PR process to make sure it passes lol

And yes, this is a real thing. Walmart's first PR check is to make sure your formatting is correct or your PR fails

Reply:
yea honostly I might like that, I need to get more into the habit of pressing ctrl s before PRs so it might help if its possible